Shared/DsaController.cpp

```diff
@@ -85,16 +86,14 @@ DsaController::DsaController(QObject* parent):
 {
   // setup config settings
   setupConfig();
+  setDefaultViewpoint();
   m_dataPath = m_dsaSettings["RootDataDirectory"].toString();
 
   connect(m_scene, &Scene::errorOccurred, this, &DsaController::onError);
 
-  // as tools are added, set the properties
-  connect(&ToolManager::instance(), &ToolManager::toolAdded, this,
-          [this](Esri::ArcGISRuntime::Toolkit::AbstractTool* tool)
+  connect(ToolResourceProvider::instance(), &ToolResourceProvider::sceneChanged, this, [this]()
   {
-    if (tool)
-      tool->setProperties(m_dsaSettings);
```

Comment:
I had a chat with @lsmallwood and addressing this would require a larger change to the app architecture. Once scenes are removed from the view they shouldn't consume as many resources, but it would be an issue for anyone loading up numerous scenes.

It's a valid concern, but won't be addressed in this PR.